tests/formattedcode/data/spdx/license_ref/expected_with_text.tv

```diff
@@ -85,10 +83,10 @@ License for ScanCode datasets
 ScanCode includes datasets (e.g. for license detection) that are dedicated
 to the Public Domain using the Creative Commons CC0 1.0 Universal (CC0 1.0)
 Public Domain Dedication: http://creativecommons.org/publicdomain/zero/1.0/</text>
-LicenseID: LicenseRef-scancode-other-permissive
-LicenseName: Other Permissive Licenses
-LicenseComment: <text>See details at https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/other-permissive.yml
+LicenseName: ScanCode acknowledgment
+LicenseComment: <text>See details at https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/scancode-acknowledgment.yml
 </text>
+LicenseID: LicenseRef-scancode-other-permissive
```

Comment:
"scancode-acknowledgment" is not "other-permissive" so there is likely a bug here.

Reply:
The order is: Id first, then name after that. So the name that belongs to this Id is "Other Permissive License".
We should probably include an empty line before the Id to better separate the different licenses.

Reply:
Ah, actually, the empty lines get stripped during the cleaning step.